docs/sphinx/conf.py

```diff
@@ -44,5 +45,5 @@
 
 intersphinx_mapping = {
     "python": ("https://docs.python.org/3", None),
-    "requests": ("https://docs.python-requests.org/en/master", None),
+    "requests": ("https://docs.python-requests.org/en/latest", None),
```

Comment:
There is a redirection in place, but `/latest` is the correct place. It also avoids a message about this redirection.